helios.js

```diff
@@ -17498,6 +17772,8 @@ class TxType extends BuiltinType {
 				return Instance.new(new TimeRangeType());
 			case "signatories":
 				return Instance.new(new ListType(new PubKeyHashType()));
+			case "redeemers":
+				return Instance.new(new MapType(new ScriptPurposeType(), new AnyDataType()));
```

Comment:
value type of the map should be `RawDataType` (which becomes `Data` inside the Helios language)